testing/apmsoak/README.md

```diff
@@ -0,0 +1,53 @@
+# Run soak test workers
+
+**NOTE: This is unlikely to be useful for non-Elastic employees**
+
+This folder provides tools to enable creation of a worker instance which have the capability to send load to any APM Server. Worker uses `apmsoak` binary to generate a continuous and sustained load. Since worker's are long running process, they are created as systemd unit with configuration to restart on failure. Monitoring is also setup for the worker as part of the terraform module to monitor the node resources and the systemd unit status.
+
+## Dependencies
+
+- `terraform`
+- `gcloud CLI`
+
+## Pre-requisites
+
+1. Install terraform
+2. Make sure that you have access to the GCP project and have enough privileges to create and manage instances. The workign set of required roles to be able to create worker using this module are:
```

Comment:
Updated the document, fixed typos. I am using the singular `worker` because the module can only create a single worker at once. Let me know if the document makes better sense this way.